texar/modules/networks/networks.py

```diff
@@ -82,3 +82,9 @@ def default_hparams():
             "layers": [],
             "name": "NN"
         }
+
+    @property
+    def output_size(self) -> int:
+        r"""The output size.
+        """
+        return self._layers[-1].out_features
```

Comment:
What if the final layer is not `nn.Linear`?

Reply:
Yeah actually the docs says `self._layers` is a list of `nn.Linear` here...